doc/dash/dash-sonic-hld.md

```diff
@@ -318,7 +318,6 @@ DASH_VNET_TABLE:{{vnet_name}}
     "vni": {{vni}} 
     "guid": {{"string"}}
     "address_spaces": {{[list of addresses]}} (OPTIONAL)
-    "peer_list": {{vnet_name_list}} (OPTIONAL)
```

Comment:
Alberto, sorry for the confusion. The entry is still present in schema as optional. However it is not used for the current scenarios. So its good to leave it here for consistency, but not to add to examples.